src/NetAnalyzers/Core/AnalyzerReleases.Shipped.md

```diff
@@ -153,7 +153,6 @@ CA2242 | Usage | Info | TestForNaNCorrectlyAnalyzer, [Documentation](https://doc
 CA2243 | Usage | Disabled | AttributeStringLiteralsShouldParseCorrectlyAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2243)
 CA2244 | Usage | Info | AvoidDuplicateElementInitialization, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2244)
 CA2245 | Usage | Info | AvoidPropertySelfAssignment, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2245)
-CA2246 | Usage | Info | AssigningSymbolAndItsMemberInSameStatement, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2246)
```

Comment:
Not sure about this change.

Reply:
@Evangelink See https://github.com/dotnet/roslyn-analyzers/pull/3694#discussion_r502807776